app/build.gradle

```diff
@@ -139,14 +139,15 @@ dependencies {
 	implementation "com.google.dagger:dagger:$daggerVersion"
 	kapt "com.google.dagger:dagger-compiler:$daggerVersion"
 
-	implementation('com.gitlab.opentransitmap:public-transport-enabler:38b99f27') {
-		exclude module: 'service'
-		exclude group: 'com.google.guava', module: 'listenablefuture'
-		exclude group: 'com.google.guava', module: 'failureaccess'
+	implementation('com.google.guava:guava:29.0-android') {
+		exclude module: 'listenablefuture'
+		exclude module: 'failureaccess'
+		exclude group: 'com.google.j2objc'
```

Comment:
Maybe we can get rid of this again, eventually. Especially if it is just some annotations, we use.

Reply:
I think most (if not all) of them are used only in Java classes, so after finishing the migration to Kotlin, we might remove the dependency.